javafx/javafx2.editor/src/org/netbeans/modules/javafx2/editor/completion/beans/FxDefinitionKind.java

```diff
@@ -47,6 +47,11 @@
      */
     LIST,
 
+    /**
+     * Readonly/immutable object, type is getter return type.
+     */
+    GETTER,
```

Comment:
This value is not used anywhere after construction - correct ?

Reply:
That is correct.